djangoproject/static/js/djangoproject.js

```diff
@@ -82,7 +82,9 @@ document.querySelector('.menu-button').addEventListener('click', function () {
 
 // Focus, select, and scroll to search input when key combination is pressed
 window.addEventListener('keydown', function (e) {
-  if (!(e.metaKey || e.ctrlKey) || e.key !== 'k') {
+  const isCtrlK = (e.metaKey || e.ctrlKey) && e.key === 'k';
+  const isSlash = e.key === '/';
+  if (!(isCtrlK || isSlash)) {
```

Comment:
I defer to the other maintainers on whether or not this change should be merged, but could you modify your code to maintain the style of the file? I think we can also factor out the `isSlash` variable:

```suggestion
  const is_ctrl_k = (e.metaKey || e.ctrlKey) && e.key === 'k';
  
  if (!(is_ctrl_k || e.key === '/')) {
```

Thanks!